wbstack/src/Settings/LocalSettings.php

```diff
@@ -525,13 +525,23 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
     }
 }
 
-# CirrusSearch etc.
-// wfLoadExtension( 'Elastica' );
-// wfLoadExtension( 'CirrusSearch' );
-// wfLoadExtension( 'WikibaseCirrusSearch' );
-// if ( $wikiInfo->getSetting('wwExtEnableWikibaseLexeme') ) {
-//     wfLoadExtension('WikibaseLexemeCirrusSearch');
-// }
+// ElasticSearch
+if ( $wikiInfo->getSetting( 'wwExtEnableElasticSearch' ) ) {
+    error_log( "elastica enabled" );
+    wfLoadExtension( 'Elastica' );
+    wfLoadExtension( 'CirrusSearch' );
+    wfLoadExtension( 'WikibaseCirrusSearch' );
+
+    // If Wikibase Lexemes are enabled, enable lexeme cirrus search
+    if ( $wikiInfo->getSetting('wwExtEnableWikibaseLexeme') ) {
+        wfLoadExtension('WikibaseLexemeCirrusSearch');
+    }
+
+    $wgSearchType = 'CirrusSearch';
+    $wgCirrusSearchServers = [ 'elasticsearch.svc' ];
```

Comment:
Server hostname should be grabbed from an environment variable.
As it needs to be configurable per deployment.

Reply:
some holiday coding? :) done!